src/System.Management.Automation/engine/Humanizer.cs

```diff
@@ -0,0 +1,132 @@
+/********************************************************************++
+Copyright (c) Microsoft Corporation.  All rights resulterved.
+--********************************************************************/
+
+using System;
+using System.Globalization;
+using System.Management.Automation;
+using System.Text;
+using System.Resources;
+using System.Reflection;
+
+namespace Microsoft.PowerShell
+{
+    /// <summary>
+    /// Contains ToHumanString CodeMethod implementations for TimeSpan
+    /// </summary>
+    public static partial class ToStringCodeMethods
+    {
+        /// <summary>
+        /// ToHumanString implementation for TimeSpan
+        /// </summary>
+        /// <param name="time">time is TimeSpan object</param>
+
+        public static string ToHumanString(TimeSpan time)
```

Comment:
Thanks for the conclusion.
I'm not a polyglot. So I needed some landmark like this `Humanizer` project. Now it is clear that each language requires _up to 4 resource strings_ and maybe their _code_. And I'm ready to forget about `Humanizer`.

I'm willing to rename things but don't feel free with English terms. I need your help.

Above I mentioned that I have not been able to compile the resource file named  Humanizer.en-Us.resx  Please comment this. Can we use existing way of working with resources here (or directly use a resource manager)? If Yes, then it requires improvements.